lib/rule_engine/parser.py

```diff
@@ -256,6 +257,10 @@ def t_DATETIME(self, t):
 		t.value = t.value[1:]
 		return t
 
+	def t_TIMEDELTA(self, t):
+		return t
+	t_TIMEDELTA.__doc__ = timedelta_regex
+
```

Comment:
So I'm thinking this should probably use a special prefix like datetime literals do. As it is now, it's pretty neat you can just define the timespan as `P1W` but that does mean that `P` can't be used as a variable which would be a breaking change.

Using a string prefix would avoid that problem, something along the lines of `p"1W1D"` for 8 days, though I'm not sure where the P comes from. I'm thinking `t` would be better since `d` maps to *d*ate / *d*atetime and `t` would map to *t*imedelta in terms of types at least.